src/bun.js/web_worker.zig

```diff
@@ -20,11 +20,9 @@ pub const WebWorker = struct {
     arena: bun.MimallocArena = undefined,
     name: [:0]const u8 = "Worker",
     cpp_worker: *anyopaque,
-    allowed_to_exit: bool = false,
     mini: bool = false,
-    parent_poll_ref: JSC.PollRef = .{},
-    initial_poll_ref: JSC.PollRef = .{},
-    did_send_initial_task: bool = false,
+    user_poll_ref: JSC.PollRef = .{},
+    eventloop_poll_ref: JSC.PollRef = .{},
```

Comment:
`parent_poll_ref`? both of these are the event loop

Reply:
i'll rename it back but the idea is one is the user's .ref/.unref, the other one is doing ref only if the worker's event loop has tasks. they both ref on the parent.

Reply:
that makes sense, the name should be clearer though